src/platform/user/authentication/utilities.js

```diff
@@ -58,7 +57,8 @@ function redirectWithGAClientId(redirectUrl) {
     const clientId = tracker && tracker.get('clientId');
 
     window.location = clientId
-      ? appendQuery(redirectUrl, { clientId })
+      ? // eslint-disable-next-line camelcase
+        appendQuery(redirectUrl, { client_id: clientId })
```

Comment:
API should expect snake_cased params as a matter of convention. It will send along to ID.me properly in camelCase, however.